### DISCUSSION TITLE
Need help to layout Divider

### DISCUSSION THREAD

**Discussion by ikus060** [beeware/toga#2429] [Q&A] [answered]
Hello, I need some help to properly layout Divider with Pack. It's not so obvious how to force widget to take all available height.

When creating vertical Divider, by default the divider only take 1px height.

![image](https://github.com/beeware/toga/assets/921251/fd648c50-5a8c-48db-a637-2e79b783d686)

```
        main_box = toga.Box(
            style=Pack(direction=COLUMN, padding=10),
            children=[
                toga.Box(
                    children=[
                        toga.Label("First section"),
                        toga.Divider(direction=toga.Divider.VERTICAL),
                        toga.Label("Second section"),
                    ],
                    style=Pack(direction=ROW, padding=10),
                )
            ]
        )
```

What I'm expecting is as follow, where divider take the same height of the siblings labels:
![image](https://github.com/beeware/toga/assets/921251/fa4604a5-2bc7-44f2-a464-3821ec1a0510)

When I apply `flex=1` it kinda work, by the separator is taking all the height of the window. It's not what I'm expecting.

In Tkinter, this is possible using Pack geometry manager with `anchor="ns"`.

**Comment by freakboy3742** [comment, marked as answer]
What you've discovered is a known bug: #1529. Unfortunately, I can't offer a good workaround. There might be something you can do to limit the expansion by putting the row box in a box, so that the row box that is marked flexible won't expand to fill the entire window; however, you're very much in experimental workaround territory here.

**Reply by ikus060** [reply]
Ok thanks, I will track the issue
